tests/integration/features/commands_without_chserver.feature

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Suggested naming like `backup_other_commands` or `backup_misc_commands`

Reply:
Fixed

Reply:
This doesn't seem to have changed.

Reply:
Yea, sorry. Renamed in the last commit.